app/_locales/en/messages.json

```diff
@@ -2793,6 +2799,9 @@
     "message": "$1 about how imported keys work",
     "description": "$1 is a link to learn more about imported keys"
   },
+  "importAnAccount": {
+    "message": "Import an account"
+  },
   "importNFT": {
```

Comment:
Adding content to match Figma (design has been updated to align with network expansion)

<img width="693" height="619" alt="Screenshot 2025-08-27 at 11 34 41 AM" src="https://github.com/user-attachments/assets/35581aaa-f39c-4006-85f0-9afe8911bae4" />